src/components/PeriodicReports/FinancialReportCheckList.tsx

```diff
@@ -0,0 +1,57 @@
+import { Checkbox, FormControl, FormControlLabel } from '@material-ui/core';
+import React from 'react';
+import {
+  FinancialReportsQuery,
+  NarrativeReportsQuery,
+} from '../../scenes/Projects/Reports/ProjectReports.generated';
+
+const ReportGuideAction = {
+  SET_RANGE: 'SET_RANGE',
+  ADD_MANAGING_PARTNER: 'ADD_MANAGING_PARTNER',
+  SET_REPORT_FREQUENCY: 'SET_REPORT_FREQUENCY',
+};
```

Comment:
This doesn't add much value, after enumerating we still check each value and do individual logic based on it. `checkList` could be simplified without it.